src/org/labkey/test/tests/LinkedSchemaInheritedPermissionsTest.java

```diff
@@ -0,0 +1,88 @@
+package org.labkey.test.tests;
```

Comment:
Don't add new tests to the root `tests` package. They should go into a subpackage based on the primary module being covered (probably "query" for linked schemas).
```suggestion
package org.labkey.test.tests.query;
```

Reply:
Done